android/src/main/java/com/passagereactnative/PassageReactNativeModule.kt

```diff
@@ -68,7 +66,7 @@ class PassageReactNativeModule(reactContext: ReactApplicationContext) :
   fun loginWithPasskey(promise: Promise) {
     CoroutineScope(Dispatchers.IO).launch {
       try {
-        val authResult = passage.loginWithPasskey("")
+        val authResult = passage.loginWithPasskey()
```

Comment:
This PR includes some very minor updates from native Android SDK (like this one) that will not impact RN developers.